sources/web/datalab/polymer/index.html

```diff
@@ -20,7 +20,7 @@
           content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 
     <title>Google Cloud Datalab</title>
-    <link rel="shortcut icon" href="images/favicon.ico">
+    <link rel="shortcut icon" href="/images/favicon.ico">
```

Comment:
Done. I'll figure out some other way to test the elements with proper paths. Maybe we'll need to put the tests along with elements.